src/tools/rbd_mirror/group_replayer/Replayer.cc

```diff
@@ -825,6 +863,8 @@ void Replayer<I>::handle_create_mirror_snapshot(
       m_local_group_snaps.erase(local_snap);
     }
   } else {
+    m_snapshot_start = ceph_clock_now();
```

Comment:
If this internal group replayer is restarted in the middle of syncing a group snapshot, where do I reset `m_snapshot_start`?

Reply:
Should we include a timestamp field in the cls struct?